docs/Deploying-Pega-on-AKS.md

```diff
@@ -501,7 +501,7 @@ helm inspect values pega/pega > <local filepath>/aks-demo/pega.yaml
+    | tier.name: ”web” tier.service.tls:| Set this parameter as `true` to encrypt the traffic between the load balancer/ingress and pods. Beginning with Helm version `2.2.0` Pega provides a default self-signed certificate; Pega also supports specifying your own CA certificate. [Also check this](#changes-in-backend-setting-if-tiersservicetlsdomain-is-true)  | <ul><li>tier.service.tls.enabled: set to `true` to enable the traffic encryption </li><li>tier.service.tls.port: 443</li><li>tier.service.tls.targetPort: 8443</li><li>tier.service.tls.external_secret_name: The external secret name for fetching certificates from a secure location. For details, see [this option in the Pega Helm chart](../charts/pega#optional-support-for-providing-credentialscertificates-using-external-secrets-operator)</li><li>tier.service.tls.keystore: The base64 encoded content of the keystore file. Leave this value empty to use the default, Pega-provided self-signed certificate. The CN(Common Name) of the certificate should match that of the domain specified in `tiers.web.ingress.domain`, see [End-to-end TLS with the v2 SKU](https://docs.microsoft.com/en-us/azure/application-gateway/ssl-overview#end-to-end-tls-with-the-v2-sku).</li><li>tier.service.tls.keystorepassword: the password of the keystore file</li><li>tier.service.tls.cacertificate: the base64 encrypted content of the root CA certificate. Leave this value empty for AKS deployments. In alternate tomcat configuration, use this field to pass the `certificateChainFile`.</li><li>tier.service.tls.certificateFile:The content of the file that contains the server certificate. The format is PEM-encoded. This should be given if keystore and keystorepassword is not specified. </li><li>service.tls.certificateKeyFile: The content of the file that contains the server private key. The format is PEM-encoded. This should be given if keystore and keystorepassword is not specified.</li><li>tier.service.traefik.enabled: set to `false` as this option is for `k8s` provider not for `AKS`</li></ul> |
```

Comment:
change 
[Also check this](#changes-in-backend-setting-if-tiersservicetlsdomain-is-true) >>
For details see [backend changes](#changes-in-backend-setting-if-tiersservicetlsdomain-is-true)

In alternate tomcat configuration,>>
For an alternate tomcat configuration that avoids using a keystore, 

to avoid redundancy, remove both> This should be given if keystore and keystorepassword is not specified. 

the rest looks pretty good.

Reply:
done